test/mongoose_riak_SUITE.erl

```diff
@@ -1,55 +0,0 @@
--module(mongoose_riak_SUITE).
```

Comment:
Why exactly was tmongoose_riak_SUITE removed?

Reply:
It shouldn't have bee committed in the first place, turns out was not ready for serious usage (does not support ssl, which is required in our Travis setup). Which basically means you were right when you said it should be a big test...

Reply:
Ok, fair enough.